src/sqlfluff/dialects/dialect_redshift.py

```diff
@@ -57,6 +60,17 @@ class DatePartFunctionNameSegment(BaseSegment):
     match_grammar = OneOf("DATEADD", "DATEDIFF")
 
 
+@redshift_dialect.segment(replace=True)
+class DatatypeIdentifierSegment(BaseSegment):
+    """A scalar or aggregate function.
+
+    Revert back to the ANSI definition to support ignore nulls
+    """
+
+    type = "function"
+    match_grammar = ansi_dialect.get_segment("FunctionSegment").match_grammar.copy()
+
+
```

Comment:
This is wrong, please remove. Provided description of DatatypeIdentifierSegment in earlier comment.